lib/openhab/dsl/types/datetime.rb

```diff
@@ -0,0 +1,339 @@
+# frozen_string_literal: true
+
+require 'java'
+require 'forwardable'
+require 'time'
+
+module OpenHAB
+  module DSL
+    module Types
+      #
+      # Ruby implementation for OpenHAB DateTimeType
+      #
+      # @author Anders Alfredsson
+      #
+      # rubocop: disable Metrics/ClassLength
+      # Disabled because this class has a single responsibility, there does not appear a logical
+      # way of breaking it up into multiple classes
+      class DateTime
+        extend Forwardable
+        include Comparable
+        include OpenHAB::Log
+
+        def_delegator :datetime, :to_s
+        def_delegator :zoned_date_time, :month_value, :month
+        def_delegator :zoned_date_time, :minute, :min
+        def_delegator :zoned_date_time, :second, :sec
+        def_delegator :zoned_date_time, :nano, :nsec
+        alias inspect to_s
+
+        java_import org.openhab.core.library.types.DateTimeType
```

Comment:
Can you switch this to the format of he other imports so it is consistent? 
Java::OrgOpenhabCore....